server/src/routes/clientRouter.js

```diff
@@ -8,7 +8,8 @@ const clientRoutes = [
   "/users/new", 
   "/restaurants", 
   "/restaurants/:id", 
-  "/burgers/:id"
+  "/burgers/:id",
+  "/reviews/:id"
```

Comment:
Since we don't have a show page for reviews, there is no need to have `/reviews/:id` in the client routes